src/index.ts

```diff
@@ -489,6 +455,15 @@ export function ConfirmMultisigAddress({
   name,
   policyHmac,
   walletConfig,
+}: {
+  keystore: KEYSTORE_TYPES;
+  network: Network;
+  bip32Path: string;
+  multisig: any;
```

Comment:
I want to retire this object so bad. It has so many different shapes, it gets mutated, the name is confusing. Just a note for the future since this one has to be `any` for now because of this confusion (or doesn't have to be but it's too complex to address now). 

Small improvement could at least instead be:

```suggestion
  multisig: Record<string, unknown>;
```

since it has to be an object.

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff